tests/integration/document-verify.test.ts

```diff
@@ -95,7 +96,7 @@ describe("POST /", () => {
       .expect(200);
 
     expect(response.body.summary).toStrictEqual(
-      RESPONSE_VERIFY_SUCCESS_SUMMARY,
+      RESPONSE_VERIFY_SUCCESS_SUMMARY
```

Comment:
as discussed, do rebase for prettier formatting check. will take a look again next week